compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/interop/UIKitInteropContainer.uikit.kt

```diff
@@ -89,3 +181,13 @@ internal fun Modifier.trackUIKitInterop(
     container = container,
     nativeView = view
 )
+
+internal inline fun <T> NSLock.doLocked(block: () -> T): T {
```

Comment:
Unused (and a bit out of scope for this file)

Reply:
It's used in other place, but what's a good place to put extensions like that?

Reply:
private method in place where it's used or some utils/lock etc. Anyway, it shouldn't be here